### ISSUE TITLE
unable to install plugin in Eclispe Mars.2 (4.5.2) Build id: M20160212-1500

### ISSUE THREAD

**Issue by souptikji** [fornwall/eclipsecoder#11]
Error is shown -
`An error occurred while collecting items to be installed
session context was:(profile=epp.package.jee, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Artifact not found: http://fornwall.net/eclipsecoder/features/net.fornwall.eclipsecoder_0.3.12-feature.jar.
http://fornwall.net/eclipsecoder/features/net.fornwall.eclipsecoder_0.3.12-feature.jar`

**Comment by fornwall** [comment]
http://fornwall.net/eclipsecoder/features/net.fornwall.eclipsecoder_0.3.12-feature.jar hasn't existed for quite some time, the latest version is http://fornwall.net/eclipsecoder/features/net.fornwall.eclipsecoder_0.3.14-feature.jar.

Can you try clearing the cache, perhaps by passing the `-clean` flag to eclipse when launching?

**Comment by souptikji** [comment]
Thanks for replying fornwall. Well I tried all possible things with Eclispe Mars.2 build, like clearing the cache (-clean flag), deleting the existing instance then re-installing the same version with a fresh workspace and all. But didn't solve the problem.
After that I went for Eclipse Luna version- the plugin works like a charm in that. So I guess some problem with the Mars build specifically. 
But anyways thanks a lot for replying :) And great job !! Helps us budding coders a lot- you guys have created an excellent product.
